pom.xml

```diff
@@ -132,7 +132,7 @@
                     <version>${maven-release-plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <releaseProfiles>javadoc,deploy,release,docker-push</releaseProfiles>
+                        <releaseProfiles>javadoc,deploy,release,docker-push,-console</releaseProfiles>
```

Comment:
There are two ways to disable a profile "!" and "-", did you check that this is supported by the plugin?